project/AkkaBuild.scala

```diff
@@ -31,6 +31,10 @@ object AkkaBuild {
       version in ThisBuild := "2.5-SNAPSHOT"
     )
 
+  lazy val strictSettings = Seq(
```

Comment:
Hmm, wouldn't this belong in `project/AkkaDisciplinePlugin`?

Reply:
Yes, that is where it belongs.

Reply:
I hadn't seen that plugin, I'll just change it to have modules where it doesn't remove the fatal wanrings